java/test/apps/gui3/FirstTimeStartUpWizardActionTest.java

```diff
@@ -1,24 +1,36 @@
 package apps.gui3;
 
-import java.awt.GraphicsEnvironment;
-
 import jmri.util.JUnitUtil;
 
 import org.junit.jupiter.api.*;
-import org.junit.Assert;
-import org.junit.Assume;
+import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
 
 /**
  *
  * @author Paul Bender Copyright (C) 2017
  */
+@DisabledIfSystemProperty(named ="java.awt.headless", matches ="true")
 public class FirstTimeStartUpWizardActionTest {
 
     @Test
     public void testCTor() {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         FirstTimeStartUpWizardAction t = new FirstTimeStartUpWizardAction("test ation");
-        Assert.assertNotNull("exists", t);
+        Assertions.assertNotNull(t, "exists");
+        t.dispose();
+    }
+
+    @Test
+    public void testNotPartOfAPanel() {
+        FirstTimeStartUpWizardAction t = new FirstTimeStartUpWizardAction("test NotPartOfAPanel", (new jmri.util.JmriJFrame("testNotPartOfAPanel Frame")));
+        Exception ex = Assertions.assertThrows(IllegalArgumentException.class, () -> { t.makePanel(); });
+        Assertions.assertNotNull(ex);
+        t.dispose();
+    }
+
+    @Test
+    public void testFirstTimeStartUpWizardAction3PartCTor() {
+        FirstTimeStartUpWizardAction t = new FirstTimeStartUpWizardAction("test ation", null, null);
```

Comment:
Just checking.
Should "test ation" be "test action"?

Reply:
thanks, typo.